src/main/java/com/petfinder/controller/UserController.java

```diff
@@ -19,44 +19,39 @@
 @Controller
 public class UserController {
 
-    @Autowired
-    UserService userservice;
-
-    @RequestMapping(value = "/register", method = RequestMethod.GET)
-    public String registerView(Model model) {
-        return "register";
-    }
-
-    @RequestMapping(value = "/register", method = RequestMethod.POST)
-    public String userAdd(@RequestParam(required = false) String login,
-                          @RequestParam(required = false) String password,
-                          @RequestParam(required = false) String repeatPassword,
-                          @RequestParam(required = false) String email,
-                          Model model) {
-        model.addAttribute("login", login);
-        model.addAttribute("email", email);
-
-        if (login.equals("") || password.equals("")
-                || repeatPassword.equals("") || email.equals("")) {
-            model.addAttribute("status", "Fields cannot remain empty.");
-            return "register";
-        }
-
-        try {
-            userservice.register(login, password, repeatPassword, email);
-        } catch (LoginExistsException | EmailExistsException
-                | PasswordsDoesNotMatchException
-                | InvalidEmailException e) {
-            model.addAttribute("status", e.getMessage());
-            return "register";
-        }
-        return "registerSuccess";
-    }
-    
+	@Autowired
+	UserService userservice;
+
+	@RequestMapping(value = "/register", method = RequestMethod.GET)
+	public String registerView(Model model) {
+		return "register";
+	}
+
+	@RequestMapping(value = "/register", method = RequestMethod.POST)
+	public String userAdd(@RequestParam(required = false) String login, @RequestParam(required = false) String password,
+			@RequestParam(required = false) String repeatPassword, @RequestParam(required = false) String email,
+			Model model) {
+		model.addAttribute("login", login);
+		model.addAttribute("email", email);
+
+		if (login.equals("") || password.equals("") || repeatPassword.equals("") || email.equals("")) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/matikz1110/PetFinder/file/2734139538/issues/source?bid=3239464&fileBranchId=3351039#l37)